ethstaker_deposit/utils/click.py

```diff
@@ -140,3 +145,19 @@ def choice_prompt_func(prompt_func: Callable[[], str], choices: Sequence[str]) -
                 output = output + ', '
     output = output + ']'
     return lambda: '%s %s: ' % (prompt_func(), output)
+
+
+def deactivate_prompts_callback(param_names: list[str]) -> Callable[[click.Context, str, str], Any]:
+    def callback(ctx: click.Context, param: Any, value: str) -> Any:
+        if value:
+            for p in ctx.command.params:
+                if isinstance(p, click.Option) and p.name in param_names and p.prompt is not None:
+                    p.prompt = None
+        else:  # CLIRunner() is not as isolated as it should be. Restore the language prompt during tests
```

Comment:
Do you think this work around is worth it versus providing the mnemonic as an argument to the `existing-mnemonic` call in the test?

Or do you think this issue could pop up again and better to resolve now versus getting hit by it later?

Reply:
The workaround is a nop outside of our unit tests. It could pop up somewhere else during unit testing, so best to nip it in the bud.

if we feel proactive we could do a minimal reproduction and raise an issue in the click GitHub, and remove this workaround if/when click fixes the issue upstream